src/governance-classes.cpp

```diff
@@ -598,18 +638,34 @@ bool CSuperblock::IsValid(const CTransaction& txNew)
     // superblock payments and the payments actually in the block, after
     // skipping any initial miner payments.
 
-    if(nMinerPayments<0)  {
+    if(nMinerPayments < 0) {
```

Comment:
:+1: My preferred coding style actually uses much more whitespace than is used here, I guess I was over-compensating.